package.yaml

```diff
@@ -298,7 +298,7 @@ executables:
     - condition: ! '!(flag(disable-git-info))'
       cpp-options: -DUSE_GIT_INFO
       dependencies:
-      - githash
+      - githash >= 0.1.2
```

Comment:
We omit version constraints from `package.yaml` since they're implied by `stack*.yaml`.  OK to leave for now though, since we do a sweep looking for any unnecessary constraints before release.